Jenkinsfile

```diff
@@ -33,14 +33,15 @@ podTemplate(yaml: readTrusted('KubernetesPod.yaml'), workingDir: '/home/jenkins/
   }
 }
 
-nodeWithTimeout('azure && docker') {
+nodeWithTimeout('docker') {
```

Comment:
quick note: if you want more resources on the VM, you can use `docker-highmem` (16 vCPUS / 64 Gb instead of the 4vCPUS / 16 Gb for `docker`).

Reply:
Thank you for the advice. I might explore that if I want to try running the tests in parallel at some point.